src/box/box.cc

```diff
@@ -641,18 +646,39 @@ wal_stream_abort(struct wal_stream *stream)
  * checking even in release.
  */
 static bool
-wal_stream_has_tx(const struct wal_stream *stream)
+wal_stream_has_tx_in_progress(const struct wal_stream *stream)
 {
 	bool has = stream->tsn != 0;
 	assert(has == (in_txn() != NULL));
 	return has;
 }
 
+/**
+ * The function checks that the tail of the transaction from the journal
+ * has been read.
+ */
+static bool
+wal_stream_has_unfinished_tx(const struct wal_stream *stream)
+{
+	bool has = wal_stream_has_tx_in_progress(stream);
+
+	bool unfinished = false;
```

Comment:
There needs to be a comment explaining why "in progress" != "unfinished".

Reply:
Done.

Reply:
From the comment I still couldn't understand it. Is "unfinished" tx the one, which has all the data, but simply isn't committed yet? Or what?

Reply:
Fixed.